bin/check-puppet-last-run.rb

```diff
@@ -95,7 +107,11 @@ def run
       @message += " with #{@failures} failures"
     end
 
-    if @now - @last_run > config[:crit_age] || @failures > 0
+    if @restart_failures > 0
```

Comment:
how does that change that we want to check if the user specified that they want to report on restart failures?